src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -681,6 +682,56 @@ bool DeformableConvolution::isSupportedOperation(const std::shared_ptr<const ngr
     return true;
 }
 
+namespace {
+
+struct DefConvKey {
+    std::vector<std::shared_ptr<BlockedMemoryDesc>> descVector;
+    DeformableConvolution::DefConvAttr defConvAttr;
+    impl_desc_type implType;
+
+    size_t hash() const;
+    bool operator==(const DefConvKey& rhs) const;
+};
+
+size_t DefConvKey::hash() const {
+    using namespace dnnl::impl;
+    using namespace dnnl::impl::primitive_hashing;
+
+    size_t seed = 0;
+
+    for (const auto& ptr : descVector) {
+        if (ptr) {
+            seed = get_vector_hash(seed, ptr->getBlockDims());
```

Comment:
Do we 100 percent sure that block dims are enough. Looks like the executor constructors use other tensor descriptor parameters as well.

Reply:
We use all components of `descVector`. Afterwards we use strides, dilations and padding. It's enough to define necessaru promitive/

Reply:
Does not it mean we have to include the strides, dilations and padding to the key as well?

Reply:
`    seed = get_vector_hash(seed, defConvAttr.stride);
    seed = get_vector_hash(seed, defConvAttr.dilation);
    seed = get_vector_hash(seed, defConvAttr.padL);`
    A couple lines lower

Reply:
Key contains `defConvAttrs` with all parameters

Reply:
Ah, OK. Thank you.